Serialization/Serialization/Helper.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Xml.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Runtime.Serialization;
+
+namespace Serialization
+{
+    public static class Helper
+    {
+        private static XmlSerializer xmlFormatter;
+        private static BinaryFormatter binaryFormatter;
+
+        static Helper()
+        {
+            xmlFormatter = new XmlSerializer(typeof(Student));
+            binaryFormatter = new BinaryFormatter();
+            
+        }
+
+        public static void SerializeToXml(Student student, Stream stream)
```

Comment:
better pass here only file location address

Reply:
Why? What if user wants to store xml not a file, but to memory, or somewhere else?